client/login/magic-login/request-login-email-form.jsx

```diff
@@ -41,19 +62,27 @@ class RequestLoginEmailForm extends React.Component {
 
 	onSubmit = event => {
 		event.preventDefault();
-		this.props.fetchMagicLoginRequestEmail( this.props.emailAddress );
+		this.props.fetchMagicLoginRequestEmail( this.state.emailAddress );
 	};
 
 	render() {
 		const {
 			currentUser,
 			requestError,
 			isFetching,
-			isValidEmailAddress,
 			emailRequested,
+			showCheckYourEmail,
 			translate,
 		} = this.props;
 
+		if ( showCheckYourEmail ) {
+			this.props.recordPageView( '/log-in/link', 'Login > Link > Emailed' );
```

Comment:
Shouldn't we move this call inside `<EmailedLoginLinkSuccessfully />` instead (similarly to what was done with `<EmailedLoginLinkExpired />`?